pkg/vulnsrc/ghsa/ghsa.go

```diff
@@ -114,7 +114,9 @@ func (vs VulnSrc) commit(tx *bolt.Tx, ghsas []GithubSecurityAdvisory) error {
 				va.FirstPatchedVersion.Identifier = strings.TrimPrefix(va.FirstPatchedVersion.Identifier, "< ")
 			}
 
-			pvs = append(pvs, va.FirstPatchedVersion.Identifier)
+			if len(va.FirstPatchedVersion.Identifier) != 0 {
```

Comment:
Just my preference:

```suggestion
			if va.FirstPatchedVersion.Identifier != "" {
```

`len(x) != 0` can be also used for slice. `x != ""` is used only for string. It is more intuitive for me.